lua/nullls.lua

```diff
@@ -18,7 +18,14 @@ nullls.setup({
 		require("null-ls").builtins.diagnostics.eslint,
 		require("null-ls").builtins.diagnostics.cpplint,
 		require("null-ls").builtins.code_actions.eslint,
-		require("null-ls").builtins.diagnostics.pylint,
+		require("null-ls").builtins.diagnostics.ruff.with({
+			command = "ruff",
+			args = { "--format", "text", "--stdin-filename", "$FILENAME", "-" },
```

Comment:
this one is the old way of using ruff
maybe we can leave as default

Reply:
Updated this part, rebased `origin/main`, and pushed with `--force-with-lease` again.